.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @grafana/grafana-oncall-backend
```

Comment:
sorting this out with IT:
<img width="1080" alt="Screenshot 2024-06-19 at 17 01 16" src="https://github.com/grafana/amixr-api-go-client/assets/9406895/36b4827a-84fe-42bb-afa4-c96bc4aba282">